new/PRS14567Dastaw.xml

```diff
@@ -0,0 +1,66 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14567Dastaw" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Dastāw</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="AY"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="AY" when="2024-09-19">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ደስታው፡ </persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Dastāw</persName>
+                    
+                    <persName xml:lang="gez" xml:id="n2" type="baptismal">ገብረ፡ ሕይወት፡</persName>
+                    <persName xml:lang="gez" corresp="#n2" type="normalized">Gabra Ḥəywat</persName>
+                    
+                    <faith type="EOTC"/>
+                    <nationality type="Ethiopia"/>
+                    <birth when="1998">1990 E.C and he was born at
+                        <placeName ref="LOC2385DabraT">Dabra Tābor</placeName></birth>
```

Comment:
He was born in 1990 E.C. in <placeName ref="LOC2385DabraT">Dabra Tābor</placeName>.